percy/snapshot.py

```diff
@@ -87,16 +87,21 @@ def percy_automate_screenshot(driver, name, **kwargs):
     if not is_percy_enabled(): return
 
     try:
+        ignore_region_elements = \
+            get_element_id_from_element(kwargs.get("ignore_region_selenium_elements", []))
+        kwargs.pop("ignore_region_selenium_elements", None)
+
```

Comment:
Can we split this in multiple lines? Having `\` if not really necessary reads bad